### PR TITLE
Chapter 5: Filling in the layout

### PR DESCRIPTION
**Pull request by maihoangminhbk**
Screenshot from 2022-08-04 16-25-14
- Change Layout
- Add user

### CODE REVIEW

**Comment by maihoangminhbk** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,17 +2,15 @@
 <html>
   <head>
     <title><%= full_title yield :title %></title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <%= render 'layouts/rails_default' %>
+    <%= render 'layouts/shim' %>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb]: dùng "

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -2,17 +2,15 @@
...
   </head>
   <body>
     <%= render "layouts/header" %>
-    
+
     <div class="container">
       <%= yield %>
+      <%= render 'layouts/footer' %>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb]: .

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (manhnt-0960) [inline app/helpers/users_helper.rb]: xóa file này vì k dùng

**Comment by maihoangminhbk** [comment]
Screenshot from 2022-08-04 17-24-50
- Update bracket format
- Delete unused files

**Comment by maihoangminhbk** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 4]
```diff
@@ -22,6 +22,31 @@ textarea {
...
+  text-align: center;
+}
+
+h2 {
+  font-size: 1.2em;
+  letter-spacing: -1px;
+  margin-bottom: 30px;
+  text-align: center;
+  font-weight: normal;
+  color: #777;
```
Comment (ghost) [inline app/assets/stylesheets/custom.scss]: đặt biến cho màu đi em

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+    <a href="https://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to t("static_pages.about.about"), static_pages_about_path %></li>
+      <li><%= link_to t("static_pages.contact.contact"), static_pages_contact_path %></li>
+      <li><a href="https://news.railstutorial.org/"><%= t("global.footer.news") %></a></li>
```
Comment (ghost) [inline app/views/layouts/_footer.html.erb]: DÙng link_to đi em

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -7,5 +7,8 @@
     <%= link_to "Ruby on Rails Tutorial", "http://www.railstutorial.org/" %>
     <%= t ".intro2" %>
   </h2>
-  <%= link_to t(".signup"), '#', class: "btn btn-lg btn-primary" %>
+  <%= link_to t(".signup"), users_new_path, class: "btn btn-lg btn-primary" %>
 </div>
+
+<%= link_to image_tag("rails.png", alt: "Rails logo", width: "200px"),
```
Comment (ghost) [inline app/views/static_pages/home.html.erb]: đưa style vào scss

File: app/views/users/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<% provide(:title, t(".signup")) %>
```
Comment (ghost) [inline app/views/users/new.html.erb]: bỏ ()

**Review by ghost — COMMENTED** [review]
cmt

**Comment by maihoangminhbk** [comment]
Screenshot from 2022-08-05 10-00-29
- Add color variables in scss
- Move style from erb to scss
- Update bracket format

**Comment by maihoangminhbk** [comment]
Screenshot from 2022-08-05 10-00-29
- Add color variables in scss
- Move style from erb to scss
- Update bracket format

**Comment by maihoangminhbk** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**